src/audio/detect_test.c

```diff
@@ -4,16 +4,29 @@
 //
 // Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
 
-#include <stdint.h>
-#include <stddef.h>
-#include <sof/spinlock.h>
-#include <sof/list.h>
-#include <sof/stream.h>
-#include <sof/ipc.h>
-#include <sof/notifier.h>
+#include <ipc/control.h>
```

Comment:
Why there are more headers included? Is it because they are needed and were indirectly included by other headers before adding them here?

Reply:
Exactly. Our code was in such a mess, that basically removing one unneeded include was causing compilation errors in ten different files.